docs/usage.md

```diff
@@ -3,64 +3,63 @@
 
 # Usage
 
-Following are the methods to use FortiSOAR Health Assessment
-- Configure FortiSOAR Health Assessment Connector
-- You can use below steps to generate record
-    - **Usage 1**: Health Assessment
-        - Navigate to Health Assessment.
-        - Click on **Assess System Health**.
-        - A Health Assessment Record will be generated. [Click here](#health-assessment) for more details.
-    - **Usage 2**: Configuration Recommendation
-        - Navigate to Health Assessment.
-        - Click on **Recommend System Configuration**.
-        - A Configuration Recommendation Record will be generated. [Click here](#configuration-recommendation) for more details.
+FortiSOAR Health Assessment can be used in following ways
+- **Usage 1**: Health Assessment
+    - Configure FortiSOAR Health Assessment Connector
+    - Navigate to Health Assessment
+    - Click on **Assess System Health**. 
+    - A Health Assessment Record will be generated. [Click here](#health-assessment) for more details
+- **Usage 2**: Configuration Recommendation
+    - Configure FortiSOAR Health Assessment Connector
```

Comment:
add this in a pre-req section or do not repeat. Is this usage is dependent on the first one? if yes then you will need to mention that as well